content/en/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -103,7 +103,7 @@ the ephemeral container to add as an `EphemeralContainers` list:
     "apiVersion": "v1",
     "kind": "EphemeralContainers",
     "metadata": {
-            "name": "example-pod"
+         "name": "example-pod"
```

Comment:
Thanks for taking the time to send a fix @yaodingyd! 👍 

It looks like this still doesn't quite line up with the `ephemeralContainers` indent, though. Could you change it to 8 spaces instead of 9?